### DISCUSSION TITLE
Streaming WebSockets : Invalid query string (400 Bad Request)

### DISCUSSION THREAD

**Discussion by artyomkonyaev** [deepgram/community#458] [General help] [answered]
I am using plain WebSockets (not a Deepgram SDK).

My query string:
`wss://api.deepgram.com/v1/listen?encoding=linear16&sample_rate=44100&smart_format=1&filler_words=1&profanity_filter=0&punctuate=1&language=en&tier=enhanced&model=general&alternatives=1`

Full error info:
```
HTTP/1.1 400 Bad Request
content-type: application/json
dg-error: Invalid query string.
content-length: 112
access-control-allow-credentials: true
vary: origin
vary: access-control-request-method
vary: access-control-request-headers
dg-request-id: 21998c92-6e98-452f-92b3-76db59bf7a7e
date: Tue, 21 Nov 2023 21:29:43 GMT
```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by adeschamps** [comment, marked as answer]
It looks like you're using `0`'s and `1`'s for parameters that expect booleans. The following should work:
```
 wss://api.deepgram.com/v1/listen?encoding=linear16&sample_rate=44100&smart_format=true&filler_words=true&profanity_filter=false&punctuate=true&language=en&tier=enhanced&model=general&alternatives=1
```
